src/Ladybug_Mesh Threshold Selector.py

```diff
@@ -150,7 +150,10 @@ def main(percent, operator):
                 srfBrep = rc.Geometry.Brep.CreateFromCornerPoints(rc.Geometry.Point3d(newMesh.Vertices[face.A]), rc.Geometry.Point3d(newMesh.Vertices[face.B]), rc.Geometry.Point3d(newMesh.Vertices[face.C]), rc.Geometry.Point3d(newMesh.Vertices[face.D]), sc.doc.ModelAbsoluteTolerance)
             else:
                 srfBrep = rc.Geometry.Brep.CreateFromCornerPoints(rc.Geometry.Point3d(newMesh.Vertices[face.A]), rc.Geometry.Point3d(newMesh.Vertices[face.B]), rc.Geometry.Point3d(newMesh.Vertices[face.C]), sc.doc.ModelAbsoluteTolerance)
-            areaList.append(rc.Geometry.AreaMassProperties.Compute(srfBrep).Area)
+            if type(srfBrep) != rc.Geometry.Brep:
```

Comment:
If it's not Brep then what is the type? Shouldn't we still collect the area?

Reply:
@mostaphaRoudsari ,
The CreateFromCornerPoints Rhinocommon method returns None when it isn't able to produce a valid brep from the points, which I guess happens for some types of invalid meshes that people are able to generate.  So a better practice would probably have been to have:
`if srfBrep is not None:`
However, because that method only returns a brep or a None value, this fix works.  Accordingly, I will merge it in anyway.